target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_zyxel_p-2812hnu-fx.dtsi

```diff
@@ -221,6 +147,64 @@
 	};
 };
 
+&gswip {
+	pinctrl-0 = <&mdio_pins>,
+		    <&gphy0_led0_pins>, <&gphy0_led2_pins>,
```

Comment:
This needs to be `<&gphy0_led1_pins>` instead of `<&gphy0_led0_pins>`.
Otherwise the gswip driver does not load because of a conflict with the stp driver:

```
[    0.698559] pinctrl-xway 1e100b10.pinmux: pin io5 already requested by 1e100bb0.stp; cannot claim for 1e108000.switch
[    0.707811] pinctrl-xway 1e100b10.pinmux: pin-5 (1e108000.switch) status -22
[    0.714849] pinctrl-xway 1e100b10.pinmux: could not request pin 5 (io5) from group gphy0 led0  on device 1e100b10.pinmux
[    0.725729] gswip 1e108000.switch: Error applying setting, reverse things back
[    0.732983] gswip: probe of 1e108000.switch failed with error -22
```

Reply:
excellent catch - thank you for reporting this.
I fixed it with the most recent push